### PR TITLE
pyddq

### PR DESCRIPTION
**Pull request by Gerrrr**
This PR is a solution for https://github.com/FRosner/drunken-data-quality/issues/91

In order to test pyddq with Travis, please put ddq jar to `python/drunken-data-quality.jar`

As described in `README.md`, `--driver-class-path` cli option should be used instead of `--packages` because of https://issues.apache.org/jira/browse/SPARK-5185. I tried to make use of `sc._jvm.java.lang.Thread.currentThread().getContextClassLoader().loadClass` magic, but did not succeed.

### CODE REVIEW

**Comment by FRosner** [comment]
I am starting the review now, @Gerrrr. Are you going to fix the codacy complaints?

**Comment by FRosner** [comment]
@Gerrrr why are there commits in the merge request that add the gitter badge and the logo?

**Comment by FRosner** [comment]
Also, travis is not building because the branch is out-of-date with master. Please do not simply merge but try to find out why we have duplicate commits (you did something wrong at some point?). After the merge has been done and conflicts are resolved, travis can build again.

**Comment by FRosner** [comment]
@Gerrrr the python install command seems to fail. Do we need sudo rights?

**Comment by FRosner** [comment]
@Gerrrr do I remember correctly that I needed to provide a custom SBT task that creates drunken-data-quality.jar?

**Comment by Gerrrr** [comment]
@FRosner, yes it is expected that the build puts the jar to `python/drunken-data-quality.jar`.

**Comment by FRosner** [comment]
@Gerrrr right now, `sbt assembly` will also execute all tests on the Scala side. Is this ok? Or shall we disable the tests for assembly?

File: python/README.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+pyddq
+=====
+
+Python binding to Drunken Data Quality
+
+
+Description
+===========
+
+A LONGER DESCRIPTION OF YOUR PROJECT GOES HERE...
```
Comment (FRosner) [inline python/README.rst:11]: Should we provide something here, @Gerrrr ? Or is it not used? I can copy something from the general readme.

Comment (Gerrrr) [inline python/README.rst:11]: Yes, it makes sense to fill in this description since this README will be available at PyPi when we upload the package.

**Comment by FRosner** [comment]
I squashed your commits and opened https://github.com/FRosner/drunken-data-quality/pull/103. I will close this PR.

**Pull request closed without merging**